src/python-common/ceph/deployment/drive_group.py

```diff
@@ -178,10 +178,12 @@ def __init__(self,
                  data_allocate_fraction=None,  # type: Optional[float]
                  method=None,  # type: Optional[OSDMethod]
                  crush_device_class=None,  # type: Optional[str]
+                 config=None, # type: Optional[Dict[str, str]]
```

Comment:
```suggestion
                 config=None,  # type: Optional[Dict[str, str]]
```
for flake8
`ceph/deployment/drive_group.py:181:30: E261 at least two spaces before inline comment`

Reply:
Done.